plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -221,6 +226,16 @@ public String getProjectId()
         return projectId;
     }
 
+    protected DatasetId toDatasetId(String schemaName)
+    {
+        return DatasetId.of(getProjectId(), schemaName);
+    }
+
+    protected String toSchemaName(DatasetId datasetId)
+    {
+        return datasetId.getDataset();
+    }
```

Comment:
I'm a little wondering why we need this method. There are still many `DatasetId.getDataset` usages.

Reply:
There's a single use of this method when we translate BigQuery datasets into Trino schemas. In all other places, we receive a Trino schema and do a reverse translation.